src/components/Settings.jsx

```diff
@@ -86,6 +92,44 @@ export default function Settings({ currentWallet }) {
           />
           Use {settings.useAdvancedWalletMode ? 'magic' : 'advanced'} wallet mode
         </rb.Button>
+
+        <br />
+
+        <rb.Button
+          variant="outline-dark"
+          className="border-0 mb-2 d-inline-flex align-items-center"
+          onClick={async (e) => {
+            e.preventDefault()
+            setRevealSensitiveInfo(false)
+            const { name: walletName, token } = currentWallet
+            const res = await Api.getSeed({ walletName, token })
```

Comment:
We shouldn't load the seed on every button click. When the user clicks the button to hide the seedphrase, we don't want to load it again. We want to  load it only when the button is clicked to reveal the seed.